airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -665,6 +665,7 @@ abstract class BasicFunctionalityIntegrationTest(
                             // "order" is a reserved word in many sql engines
                             "order" to stringType,
                             "ProperCase" to stringType,
+                            "Foo.Bar" to stringType,
```

Comment:
this doesn't actually catch the bug (b/c the bug was specifically in dedup syncs). But it's cheap to add, so why not :shrug: